### PR TITLE
Add speaker bureau page

### PR DESCRIPTION
**Pull request by AnitaYCheng**
# Pull request summary
- Add speaker bureau page, with new image
- Update call-to-action boxes for padding and corner radius

### CODE REVIEW

**Review by melchoyce — COMMENTED** [review]
Looks good design-wise!

**Review by melchoyce — APPROVED** [review]

File: content/pages/contact/speaker-bureau.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+If you’re looking for a speaker for your upcoming event or program, 18F staff have experience speaking about the following topic areas:
+
+- Engineering, including modernization, scaling, security, and Agile practices
+- Design, including UX design, user research, service design, information architecture, and content strategy
+- Product management in government
+- Stakeholder management
+- Government acquisition
+
+Formats including conference talks, workshops, and small group events.
```
Comment (beechnut) [inline content/pages/contact/speaker-bureau.md:19]: thing to consider: I know there's a call-to-action on the side, but, when I get to this end sentence, I'm finding myself wanting another sentence with a prose call to action, like a simple link.

File: assets/img/speaking/ron-speaking-color-corrected.jpg
Comment (beechnut) [inline assets/img/speaking/ron-speaking-color-corrected.jpg:1]: compliment: This edit highlights Ron much better! Nice work :)

File: content/pages/contact/contact.md [hunk 2]
```diff
@@ -13,6 +13,9 @@ Contact our business development team at [inquiries18F@gsa.gov](mailto:inquiries
 ### Looking for a job at 18F?
 18F is part of GSA's Technology Transformation Services (TTS). View all [TTS open positions](https://join.tts.gsa.gov/) or email the talent team at [joinTTS@gsa.gov](mailto:joinTTS@gsa.gov).
 
+### Want to have 18F speak at your event?
+Check out [topics we can give talks about]({{ "/speaker-bureau/" |url }}).
```
Comment (beechnut) [inline content/pages/contact/contact.md:17]: concern: My understanding is that, when screen readers scan links, they read the link copy.

change: I think the full sentence is clearer than the link fragment, so I'd suggest making the entire sentence the link.

File: templates/email-cta.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<div class="bg-primary-darker padding-3 radius-md">
+  <p class="font-sans-lg text-medium text-white">
+    Contact our outreach team to have 18F speak at your event.
+  </p>
+  <div>
+    <a href="mailto:18F@gsa.gov" class="usa-button an18f-button--accent"
```
Comment (beechnut) [inline templates/email-cta.html:6]: suggestion (non-blocking):

```html
<a href="mailto:18F@gsa.gov" class="usa-button an18f-button--accent">
  {% image "assets/img/send-2--c.svg" "" %}
  Email us
</a>
```

File: templates/side-cta.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<div class="bg-primary-darker margin-y-4 padding-2">
+<div class="bg-primary-darker margin-y-4 padding-3 radius-md">
```
Comment (beechnut) [inline templates/side-cta.html:1]: compliment: I love how this change looks, both the padding and radius.

File: content/pages/contact/contact.md (hunk 2 above)
Comment (beechnut) [inline content/pages/contact/contact.md:17]: change: Add a space between `|` and `url`

**Review by beechnut — CHANGES_REQUESTED** [review]
Wow, this looks great! And I appreciate all the changes you made based off of the last PR's comments.

I think I only see two minor changes to make — the rest of the comments are non-blocking suggestions and compliments.

**Pull request merged**